### DISCUSSION TITLE
anki to quizlet not working

### DISCUSSION THREAD

**Discussion by jbish90** [sviatoslav-lebediev/anki-quizlet-importer-extended#172] [Q&A]
can anyone help me as unable to transfer my quilzet cards to anki keeps showing an unknown error for this card set 
https://quizlet.com/gb/888939703/quizlet-theme-1-business-flash-cards/?funnelUUID=b32786a8-ca24-4613-af74-08a39364417a

**Comment by jbish90** [comment]
would be really helpful as need to do more sets

**Comment by sviatoslav-lebediev** [comment]
here is your deck. but next time please provide more info about your problem.

[QUIZLET THEME 1 BUSINESS.apkg.zip](https://github.com/sviatoslav-lebediev/anki-quizlet-importer-extended/files/14551483/QUIZLET.THEME.1.BUSINESS.apkg.zip)
